Vagrantfile

```diff
@@ -11,17 +12,20 @@ Vagrant.configure("2") do |config|
     ]
   end
 
-  config.vm.define "appserver"
+  config.vbguest.auto_update = false
```

Comment:
What does this do?

Reply:
every boot, at least for my machine, tries to upgrade (and fail) the guest additions from 7.0.6 to 7.0.8, so I am trying to install it at the first bootstrap (see start.sh) and never again